### ISSUE TITLE
Incompatablity with Creatures from the snow

### ISSUE THREAD

**Issue by Eatham532** [fxmorin/carpet-fixes#118]
https://www.curseforge.com/minecraft/mc-mods/creatures-from-the-snow-fabric

When having carpet fixes installed my game won't start. I get the following in my log. Nothing comes up as a crash report. 

``` @Redirect conflict. Skipping debugify.mixins.json:basic.mc160095.CactusBlockMixin from mod debugify->@Redirect::replaceBlockState(Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680; with priority 1000, already redirected by carpet-fixes.mixins.json:blockFixes.CactusBlock_movingBlockMixin from mod carpet-fixes->@Redirect::getCorrectBlockStateIfMovingPiston(Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680; with priority 1000
@Redirect conflict. Skipping carpet-fixes.mixins.json:blockFixes.EnchantingTableBlock_transparentMixin from mod carpet-fixes->@Redirect::isTranslucent(Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)Z with priority 1000, already redirected by easymagic.common.mixins.json:EnchantmentTableBlockMixin from mod easymagic->@Redirect::isEmptyBlock(Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)Z with priority 1001
Console18:23:35
at net.minecraft.class_5458.method_44104(class_5458.java:98) ~[server-intermediary.jar:?]
at net.minecraft.class_5458.method_30566(class_5458.java:105) ~[server-intermediary.jar:?]
at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
at net.minecraft.class_5458.<clinit>(class_5458.java:104) ~[server-intermediary.jar:?]
at net.minecraft.class_2378.<clinit>(class_2378.java:326) ~[server-intermediary.jar:?]
at net.minecraft.class_2966.method_12851(class_2966.java:50) ~[server-intermediary.jar:?]
at net.minecraft.server.Main.main(Main.java:98) ~[server-intermediary.jar:?]
at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.10.jar:?]
... 4 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: @At("INVOKE") on net/minecraft/class_5635::powderSnowNotJustFeet with priority 1000 cannot inject into net/minecraft/class_5635::method_9548(Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_1297;)V merged by net.wabbitstudios.creaturesofthesnow.mixin.PowderSnowMixin with priority 1000 [PREINJECT Applicator Phase -> carpet-fixes.mixins.json:blockFixes.PowderSnowBlock_feetAndCenterMixin from mod carpet-fixes -> Prepare Injections -> -> redirect$bmd000$powderSnowNotJustFeet(Lnet/minecraft/class_1297;)Lnet/minecraft/class_2680; -> Prepare]
at org.spongepowered.asm.mixin.injection.code.Injector.findTargetNodes(Injector.java:305) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.injection.code.Injector.find(Injector.java:240) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.prepare(InjectionInfo.java:421) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1338) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1043) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:393) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:414) ~[fabric-loader-0.14.10.jar:?]
at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.14.10.jar:?]
at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.10.jar:?]
at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) ~[fabric-loader-0.14.10.jar:?]
at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
at net.minecraft.class_1802.<clinit>(class_1802.java:21) ~[server-intermediary.jar:?]
at net.mehvahdjukaar.moonlight.api.fluids.FoodProvider.<clinit>(FoodProvider.java:33) ~[moonlight-1.19.2-2.1.9-fabric.jar:?]
at net.mehvahdjukaar.moonlight.api.fluids.SoftFluid$Builder.<init>(SoftFluid.java:243) ~[moonlight-1.19.2-2.1.9-fabric.jar:?]
at net.mehvahdjukaar.moonlight.api.fluids.fabric.SoftFluidRegistryImpl.getDefaultValue(SoftFluidRegistryImpl.java:81) ~[moonlight-1.19.2-2.1.9-fabric.jar:?]
at net.minecraft.class_5458.method_44104(class_5458.java:98) ~[server-intermediary.jar:?]
at net.minecraft.class_5458.method_30566(class_5458.java:105) ~[server-intermediary.jar:?]
at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
at net.minecraft.class_5458.<clinit>(class_5458.java:104) ~[server-intermediary.jar:?]
at net.minecraft.class_2378.<clinit>(class_2378.java:326) ~[server-intermediary.jar:?]
at net.minecraft.class_2966.method_12851(class_2966.java:50) ~[server-intermediary.jar:?]
at net.minecraft.server.Main.main(Main.java:98) ~[server-intermediary.jar:?]
at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.10.jar:?]
... 4 more ```

**Comment by Hahahavana** [comment]
Same mixin issue with the most recent version of [Frozen Up](https://www.curseforge.com/minecraft/mc-mods/frozen-up)

**Comment by FxMorin** [comment]
Working on it

**Comment by FxMorin** [comment]
Should be fixed in v1.13.0

**Comment by An-thracene** [comment]
Will this fix be backported to 1.19.2?

**Comment by akai-hana** [comment]
> Will this fix be backported to 1.19.2?

apparently not. sorry for necrobump, but also had a mixin incompatibility issue with frozenup.
